### PR TITLE
[Release]linux_sql all

### PR DESCRIPTION
**Pull request by Ceciliaaayang**

### CODE REVIEW

File: linux_sql/scripts/psql_docker.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# script usage
+#./scripts/psql_docker.sh start|stop|create [db_username][db_password]
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:3]: remove unused code and extra empty lines

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+enter_cmd=$1
+db_username=$2
+db_password=$3
+
+
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: do not have more than 1 empty lines

File: linux_sql/scripts/psql_docker.sh [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+#if user enter create option
+if [ "$1" == "create" ] && [ "$#" = "3" ];
+then
+	#check whether container already created
+	if [ "$(docker ps -f name= "jrvs-psql" | wc -l)" = "2" ];
+	then
+		echo "container is already created"
+       	 	exit 1
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:16]: fix the indentation please

File: linux_sql/scripts/psql_docker.sh [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+	fi
+
+        docker volume create pgdata 
+
+	#create container
+	if [ "$(docker container ls -a -f name= "jrvs-psql" | wc -l)" != "2" ];
+   	then 
+		docker run --name jrvs-psql -e POSTGRES_USER=${db_username} -e POSTGRES_PASSWORD=${db_password} -d -v pgdata:/var/lib/postgresql/data -p 5432:5432 postgres
+		echo "container created"
+		exit 0
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: use exit $?

File: linux_sql/scripts/psql_docker.sh [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+	
+
+fi
+
+
+#if user didn't enter username or password
+if [ "$1" == "create" ] && [ "$#"!=3 ];
+then 
+	echo "Invalid argument, please enter username and password"
+        exit 1
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:34]: fix all indentation issues

File: linux_sql/scripts/psql_docker.sh [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+
+
+	
+
+fi
+
+
+#if user didn't enter username or password
+if [ "$1" == "create" ] && [ "$#"!=3 ];
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: use if-else instead of two ifs

File: linux_sql/scripts/psql_docker.sh [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+	echo "Invalid argument, please enter username and password"
+        exit 1
+fi
+
+
+#if user enter start option 
+if [ "$1" == "start" ]; 
+then 
+    #start docker 
+    	systemctl status docker && systemctl start docker
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:41]: systemctl status docker || systemctl start docker
you need to know why

**Review by jarviscanada — CHANGES_REQUESTED** [review]
fix all comment and request another code review

File: linux_sql/scripts/psql_docker.sh (hunk 3 above)
Comment (Ceciliaaayang) [reply linux_sql/scripts/psql_docker.sh:16]: I am not sure why it shows in pull request this but in origin file and GitHub file , it didn't show indentation

File: linux_sql/scripts/psql_docker.sh (hunk 7 above)
Comment (Ceciliaaayang) [reply linux_sql/scripts/psql_docker.sh:41]: check the status and start docker not or, or means choose one of them

File: linux_sql/scripts/psql_docker.sh (hunk 3 above)
Comment (jarviscanada) [reply linux_sql/scripts/psql_docker.sh:16]: it's probably a GitHub glitch

File: linux_sql/scripts/psql_docker.sh (hunk 7 above)
Comment (jarviscanada) [reply linux_sql/scripts/psql_docker.sh:41]: use `||` instead of `&&`

**Review by jarviscanada — APPROVED** [review]
fix all and merge. 
don't forget to open a master PR for MCR

**Pull request merged**